rust/trace/src/chrome_trace_dump.rs

```diff
@@ -82,6 +83,36 @@ where
     serde_json::from_reader(input).map_err(Error::Json)
 }
 
+/// Save tracing data to to supplied path, using the Trace Viewer format. Trace file can be opened
+/// using the Chrome browser by visiting the URL `about:tracing`. If `sorted_chronologically` is
+/// true then sort output traces chronologically by each trace's time of creation.
+pub fn save_traces(output_path: &Path, sorted_chronologically: bool) -> IOResult<()> {
```

Comment:
I think the correct type for `output_path` is `<P: AsRef<Path>>`. This has the advantage of working with `&Path`, `&PathBuf`, `PathBuf`, `String`, `str`, `OsStr`, etc.